### PR TITLE
raidboss: cn translation for delubrum reginae triggers

### PR DESCRIPTION
**Pull request by miaonima**
Edit suggestions welcome!

### CODE REVIEW

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 1]
```diff
@@ -780,6 +809,7 @@ export default {
           en: 'Find Charge, Dodge Right',
           de: 'Halte nach dem Ansturm ausschau, weiche nach rechts aus',
           fr: 'Repérez la charge, esquivez à droite',
+          cn: '右侧躲避冲锋',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:812]:
```suggestion
          cn: '躲避右侧冲锋',
```
right?

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 2]
```diff
@@ -796,6 +826,7 @@ export default {
           en: 'Find Charge, Dodge Left',
           de: 'Halte nach dem Ansturm ausschau, weiche nach links aus',
           fr: 'Repérez la charge, esquivez à gauche',
+          cn: '左侧躲避冲锋',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:829]:
```suggestion
          cn: '躲避左侧冲锋',
```

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 3]
```diff
@@ -748,18 +774,21 @@ export default {
...
           de: 'Zu kleinen Bomben zurückstoßen lassen',
           fr: 'Faites-vous pousser sur une petite bombe',
+          cn: '击退到小的炸弹',
           ko: '작은 폭탄으로 넉백당하기',
         },
         getKnockedToLargeBomb: {
           en: 'Get Knocked To Large Bomb',
           de: 'Zu großen Bomben zurückstoßen lassen',
           fr: 'Faites-vous pousser sur une grosse bombe',
+          cn: '击退到大的炸弹',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:791]:
```suggestion
          cn: '击退到大炸弹',
```
abbrivable `的`.

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 4]
```diff
@@ -263,6 +272,7 @@ export default {
           en: 'Follow Second Charge',
           de: 'Folge dem 2. Ansturm',
           fr: 'Suivez la deuxième charge',
+          cn: '跟紧第二次冲锋',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:275]:
```suggestion
          cn: '紧跟第二次冲锋',
```

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 5]
```diff
@@ -462,12 +477,14 @@ export default {
           en: 'Go To North Circle',
           de: 'Geh zum Kreis im Norden',
           fr: 'Allez au donut Nord',
+          cn: '去北面，站月环里',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:480]:
```suggestion
          cn: '去上面(北面)月环',
```

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 6]
```diff
@@ -462,12 +477,14 @@ export default {
...
           ko: '북쪽 원으로 이동',
         },
         weaveWithKnockback: {
           en: 'Get Knocked Back To Circle',
           de: 'Lass dich zum Kreis im Norden zurückstoßen',
           fr: 'Faites-vous pousser dans le donut',
+          cn: '击退到北面月环里',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:487]:
```suggestion
          cn: '击退到上面(北面)月环中',
```

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 7]
```diff
@@ -547,6 +565,7 @@ export default {
           en: 'Stand In Opposite Meteor',
           de: 'Steh im entgegengesetztem Meteor',
           fr: 'Placez-vous au météore de l\'élément opposé',
+          cn: '吃相反温度的陨石',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:568]:
```suggestion
          cn: '接相反温度的陨石',
```

You can use 接, but avoid to use 吃 in something you **should** do. Here and everywhere.

**Review by MaikoTan — COMMENTED** [review]
Thanks for the translation, some minor comments.

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js (hunk 1 above)
Comment (miaonima) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:812]: I think this should be rephrased as "去右侧躲避冲锋". Please see a screenshot for this mechanic.
charge_right

Comment (MaikoTan) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:812]: (・・;
Ah, Okay!
Agreed "去右侧躲避冲锋"!

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 8]
```diff
@@ -748,18 +774,21 @@ export default {
...
           de: 'Zur Mitte zurückstoßen lassen',
           fr: 'Faites-vous pousser vers le milieu',
+          cn: '击退到中间',
           ko: '중앙으로 넉백 당하기',
         },
         getKnockedToSmallBomb: {
           en: 'Get Knocked To Small Bomb',
           de: 'Zu kleinen Bomben zurückstoßen lassen',
           fr: 'Faites-vous pousser sur une petite bombe',
+          cn: '击退到小的炸弹',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:784]: Also delete "的" here pls

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 9]
```diff
@@ -374,6 +387,7 @@ export default {
           en: 'Stand On Small Bomb',
           de: 'Auf kleinen Bomben stehen',
           fr: 'Placez-vous sur une petite bombe',
+          cn: '站在小的炸弹上',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:390]: Also delete "的" here pls

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae.js [hunk 10]
```diff
@@ -396,6 +410,7 @@ export default {
           en: 'Stand On Large Bomb',
           de: 'Auf großen Bomben stehen',
           fr: 'Placez-vous sur une grosse bombe',
+          cn: '站在大的炸弹上',
```
Comment (MaikoTan) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae.js:413]: Also delete "的" here pls

Comment (miaonima) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae.js]: Sure!

**Review by MaikoTan — APPROVED** [review]

**Comment by quisquous** [comment]
Thanks for the PR, and thanks for the review, Maiko!

**Pull request merged**